server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -347,13 +347,19 @@ export default class ActivitiesMigrationController {
     }
   }
 
-  private static alreadyMigratedAppInsightsQuery(migrationId: string, startedDate: string, endedDate: string): string {
+  private static fullMigrationAppInsightsQuery(migrationId: string, startedDate: string, endedDate?: string): string {
+    const startDateQuery = `datetime(${this.toISODateTime(startedDate)})`
+    const endDateQuery = endedDate ? `datetime(${this.toISODateTime(endedDate)})` : `now()`
     return `customEvents
       | where cloud_RoleName == 'hmpps-prisoner-from-nomis-migration'
-      | where timestamp between (datetime(${ActivitiesMigrationController.toISODateTime(
-        startedDate,
-      )}) .. datetime(${ActivitiesMigrationController.toISODateTime(endedDate)}))
+      | where timestamp between (${startDateQuery} .. ${endDateQuery})
       | where customDimensions.migrationId startswith '${migrationId}'
+      | where name startswith 'activity-migration'
+    `.trim()
+  }
```

Comment:
Also updated activities and allocations App Insights queries to fix a bug where the query failed if the endedDate was null (i.e. the migration is still running).